### ISSUE TITLE
GenomicsDBImport error 'Duplicate field name MMQ found in vid attribute "fields"'

### ISSUE THREAD

**Issue by tkoomar** [bcbio/bcbio-nextgen#2674]
On the latest development branch, in my fist joint calling with GATK4 I am getting the following error as soon as the joint calling starts:
```
Duplicate field name MMQ found in vid attribute "fields"
terminate called after throwing an instance of 'FileBasedVidMapperException'
  what():  FileBasedVidMapperException : Duplicate fields exist in vid attribute "fields"
' returned non-zero exit status 250
```
[Full log is at this gist](https://gist.github.com/tkoomar/15093628edf5fd9393153eb9826dbc9c)

I'm not sure if this is a bcbio issue, or something upstream in GATK, as I am not yet familiar with   GenomicsDBImport. Looking at the VCF header produced by GenomicsDBImport, there appears to be both a FORMAT and an INFO MMQ field:

```
##FORMAT=<ID=MMQ,Number=A,Type=Float,Description="median mapping quality">
##INFO=<ID=MMQ,Number=R,Type=Integer,Description="median mapping quality">
```

From the representations of the two fields, it looks like the INFO field is getting properly reformatted by `vcfutils.py` (al la ef0d1fc532d08ebaf1fbfa875f41637827c401ea) , while the FORMAT field is added afterward.

The only ["Duplicate field name" error on the GATK forums ](https://gatkforums.broadinstitute.org/gatk/discussion/13348/genomicsdbimport-duplicate-field-name-mq0-found-in-vid-map)seems to stem from the way GATKv3.5 formatted the gVCF headers, which is why I suspect the header (though all of this was obviously run on GATK4).

**Comment by chapmanb** [comment]
Tanner;
Thanks for the report and apologies about the issue. I believe that the recent version of GATK (4.1.0.0) moved MMQ from FORMAT to INFO in the annotations. Is it possible you ran parts of your project with different versions of GATK? If so, you could have some VCF inputs to joint calling with MMQ in INFO and others with it in FORMAT, causing the dual definitions and errors from GenomicsDBImport. When I run small test cases with 4.1.0.0, all of the MMQ fields are in the INFO field and it does work cleanly. Would upgrading in the middle of the run help explain what is happened? If so, the easiest approach to fix might be to remove the calls from the previous version and then re-run bcbio to recall those. Sorry, this looks like quite a big project from the command lines so not sure how much difficulty this would cause. Hope this helps explain what is happening.

**Comment by tkoomar** [comment]
Thanks for the quick reply Brad. I did an upgrade when this run failed previously because of  #2611, which is probably when GATK got updated. Looks like ~70 out of the ~400 samples have MMQ in FORMAT rather than INFO, so I suppose it could be worse. 

I have removed all of those samples' various files from `gatk-haplotype` and  its sub-folders, and am re-running now. It will take a few days to get back to joint calling, but I will close this issue if it does so successfully.
